content-repo/extra-docs/articles/setup-google-maps.md

```diff
@@ -0,0 +1,47 @@
+---
+id: google-maps
+title: Setup Google Maps in Cortex XSOAR using an Automation
+description: How to setup Google Maps in Cortex XSOAR using an automation.
+---
+
+You need to set up Google Maps in Cortex XSOAR before using either the [ShowOnMap](https://xsoar.pan.dev/docs/reference/scripts/show-on-map) or the `ShowLocationOnMap` automation. To use Google Maps, you need to create a `Google Maps Platform API` from a Google Cloud account that has [billing enabled](https://developers.google.com/maps/documentation/javascript/cloud-setup#billing). After creating the API, you need to add it to Cortex XSOAR. You can then use the `ShowOnMap` automation.
+If using the `ShowLocationOnMap` automation, to view the map, you need to add the automation to a indicator layout. 
+
+
+1. In **Google Cloud Platform**, do the following:
+   1. Create a [Google Cloud Project](https://developers.google.com/maps/documentation/javascript/cloud-setup).
+   2. Create a [Google Maps Platform API](https://developers.google.com/maps/documentation/javascript/get-api-key) for your project.
```

Comment:
Recommend calling the API only by the name Google uses in its documentation, Maps JavaScript API, to avoid confusion.